data/secscan_model/__init__.py

```diff
@@ -1,16 +1,74 @@
 import os
 import logging
+from collections import namedtuple
 
 from data.secscan_model.secscan_v2_model import V2SecurityScanner
+from data.secscan_model.secscan_v4_model import V4SecurityScanner
+from data.secscan_model.interface import SecurityScannerInterface
+from data.database import Manifest
+from data.registry_model.datatypes import Manifest as ManifestDataType
+
 
 logger = logging.getLogger(__name__)
 
 
+SplitScanToken = namedtuple("NextScanToken", ["version", "token"])
+
+
 class SecurityScannerModelProxy(object):
     def configure(self, app, instance_keys, storage):
         self._model = V2SecurityScanner(app, instance_keys, storage)
+        self._v4_model = V4SecurityScanner(app, instance_keys, storage)
+        self._v4_namespace_whitelist = app.config.get("SECURITY_SCANNER_V4_NAMESPACE_WHITELIST", [])
+
+        logger.info("===============================")
+        logger.info(
+            "Using split secscan model: v4 whitelist `%s`", self._v4_namespace_whitelist,
+        )
+        logger.info("===============================")
+
+        return self
 
     def __getattr__(self, attr):
```

Comment:
It's simpler to define the method splitting here instead of using a `SplitModel` pattern like we did with `registry_model`.

Reply:
Okay. That's fine, given how simple this interface is.